File: android/src/main/java/top/huic/tencent_im_plugin/message/entity/AbstractMessageEntity.java
package top.huic.tencent_im_plugin.message.entity;

import java.io.Serializable;

import top.huic.tencent_im_plugin.enums.MessageNodeType;

/**
 * 消息实体
 *
 * @author 蒋具宏
 */
public class AbstractMessageEntity implements Serializable {
    private MessageNodeType nodeType;

    public AbstractMessageEntity(MessageNodeType nodeType) {
        this.nodeType = nodeType;
    }

    public MessageNodeType getNodeType() {
        return nodeType;
    }

    public void setNodeType(MessageNodeType nodeType) {
        this.nodeType = nodeType;
    }
}

File: ios/Classes/message/entity/AbstractMessageEntity.swift
//
//  AbstractMessageEntity.swift
//  tencent_im_plugin
//
//  Created by 蒋具宏 on 2020/3/15.
//  抽象消息实体
public class AbstractMessageEntity : NSObject{
    var nodeType : MessageNodeType?;
    
    override init() {
    }
    
    init(_ type : MessageNodeType) {
        nodeType = type;
    }
}
